docs/src/dev-test-monitoring.md

```
@@ -0,0 +1,24 @@
+# Port-forward to access the dashboard
+kubectl port-forward -n prometheus services/prometheus-stack-grafana 3000:80
+
+# Get the admin user.
+kubectl get secret --namespace prometheus prometheus-stack-grafana -o jsonpath="{.data.admin-user}" | base64 --decode ; echo
+# Get the admin password.
+kubectl get secret --namespace prometheus prometheus-stack-grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+```
+
+After this, you can use [`./playground/infra/elasti-dashboard.yaml`](https://github.com/truefoundry/KubeElasti/blob/main/playground/infra/elasti-dashboard.yaml) to import the KubeElasti dashboard.
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)